third-party/realdds/src/dds-stream-server.cpp

```diff
@@ -107,7 +108,7 @@ dds_pose_stream_server::dds_pose_stream_server( std::string const & stream_name,
 }
 
 
-void dds_stream_server::open( std::string const & topic_name, std::shared_ptr< dds_publisher > const & publisher )
+void dds_video_stream_server::open( std::string const & topic_name, std::shared_ptr< dds_publisher > const & publisher )
```

Comment:
I think this is leftover? Seems it shouldn't be this way...

Reply:
I left different `open` implementation for video and motion because motion will soon use a different topic type.